patches.json

```diff
@@ -203,8 +203,10 @@
       ">=2.3.2 <2.3.4": "MAGECLOUD-3392__reduce_q-ty_of_error_report_files__2.3.2.patch"
     },
     "Fix pagebuilder module": {
-      "2.3.1": "MDVA-22979__fix_pagebuilder_module__2.3.1.patch",
-      "2.3.2": "MDVA-22979__fix_pagebuilder_module__2.3.2.patch"
+      "2.3.1": "PB-319__fix_pagebuilder_module__2.3.1.patch",
+      "2.3.2": "PB-320__fix_pagebuilder_module__2.3.2.patch",
+      ">=2.3.2-p1 <=2.3.2-p2": "PB-322__fix_pagebuilder_module__2.3.2-p1.patch",
```

Comment:
In other patches we were using `<=2.3.3` instead of `<=2.3.2-p2`. The reason is - if eventually it is re-packed for some reason, the patch needs to keep applying.

Reply:
@YPyltiai resolved.